applications/dashboard/js/jquery.fluidfixed.js

```diff
@@ -63,6 +64,8 @@
                     // We're going to extend past the page bottom.
                     $(element).css('margin-top', vars.maxMarginDiff);
                 }
+            } else {
```

Comment:
Fixes issue on a page when the nav is longer than the page height when expanded but shorter when collapsed. Collapsing the nav makes it fixed, but expanding it again doesn't un-fix it, so the nav gets cut off.